src/expand.cpp

```diff
@@ -17,6 +17,7 @@
 #include <procfs.h>
 #endif
 #if __APPLE__
+#include <sys/time.h> // Required to build with old SDK versions
```

Comment:
possibly make this an IWYU pragma?

Reply:
I'm not set up to run IWYU so I don't know if it would even flag this. We do use something from this file, it's just that in newer SDKs it's redundant because `<sys/proc.h>` includes it.